demos/tests/cases.py

```diff
@@ -18,6 +18,7 @@
 from args import *
 from image_sequences import IMAGE_SEQUENCES
 
+MONITORS = 'cdm'
```

Comment:
I think you can go further: do `MONITORS = {'-u': 'cdm'}` and merge it into the test cases with `**MONITORS` .

Reply:
Updated.